Diffing_Engine/Properties/launchSettings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Can we not commit this file to the repo please

Reply:
committed by mistake, thanks for noticing. Removed.